server/odc-core/src/main/java/com/oceanbase/odc/core/shared/constant/LimitMetric.java

```diff
@@ -45,7 +45,8 @@ public enum LimitMetric implements Translatable {
     EXPORT_OBJECT_COUNT,
     TABLE_NAME_LENGTH,
     WORKSHEET_CHANGE_COUNT,
-    WORKSHEET_SAME_LEVEL_COUNT;
+    WORKSHEET_SAME_LEVEL_COUNT,
+    WORKSHEET_PROJECT_COUNT;
```

Comment:
concept confused, it is not clear here mean worksheet count of project count,
may WORKSHEET_COUNT_IN_PROJECT

Reply:
Okay，I'll make this change.